docs/index.md

```diff
@@ -11,20 +11,22 @@ parent="smn_workw_compose"
 
 # Overview of Docker Compose
 
-Compose is a tool for defining and running multi-container applications with
-Docker. With Compose, you define a multi-container application in a single
-file, then spin your application up in a single command which does everything
-that needs to be done to get it running.
+Compose is a tool for defining and running multi-container Docker applications.
+With Compose, you define a multi-container application in a compose
+file then, using a single command, you create and start all the containers
+from your configuration. To learn more about all the features of Compose
+see [the list of features](#features)
 
-Compose is great for development environments, staging servers, and CI. We don't
-recommend that you use it in production yet.
+Compose is great for development, testing, and staging environments, as well as
+CI workflows. You can learn more about each case in
+[Common Use Cases](#common-use-cases).
 
 Using Compose is basically a three-step process.
 
 1. Define your app's environment with a `Dockerfile` so it can be
 reproduced anywhere.
 2. Define the services that make up your app in `docker-compose.yml` so
```

Comment:
Yeah, I think I like your use of `docker-compose.yml` rather than saying Compose file.  Though, come to think of it, too bad it isn't `Composefile` like `Dockerfile` eh?  We would be consistent.